DAS/das/utils/logger.py

```diff
@@ -156,17 +141,16 @@ def log(self, mqtt_topic: str, message: str) -> None:
         message : str
             Corresponding message to be recorded
         """
-        time_delta = time.monotonic() - self._START_TIME
+        time_delta = time.monotonic() + self._LAST_RECORDED_TIME
```

Comment:
We will only get relative time from monotonic time. This is because we do not have a real time module on the pi. Due to having it run potentially offline.